dev/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7.4-fpm
+
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
```

Comment:
Can we use node 18 as this is what we're running on the web server?

Reply:
Yes we can, I uploaded the version